common/include/kolmogorov_smirnov_impl.hpp

```diff
@@ -24,7 +24,7 @@ namespace datasketches {
 
 template<typename Sketch>
 double kolmogorov_smirnov::delta(const Sketch& sketch1, const Sketch& sketch2) {
-  using Comparator = typename Sketch::comparator;
+  auto comparator = sketch1.get_comparator(); // assuming the same comparator in sketch2
```

Comment:
should we care if sketch1 and sketch2 have different comparators?

Reply:
right now they are passed as a type, not instance. sketch1 and sketch2 have the same type. potentially we could have two template parameters to support sketches of different types. I am not sure we need this